src/main/java/org/folio/innreach/domain/service/impl/InnReachTransactionActionServiceImpl.java

```diff
@@ -317,7 +317,7 @@ public void returnPatronHoldItem(UUID transactionId, UUID servicePointId) {
   public void transferItemHold(UUID transactionId, UUID itemId) {
     var transaction = fetchTransactionOfType(transactionId, ITEM);
 
-    verifyState(transaction, ITEM_HOLD);
+    verifyState(transaction, ITEM_HOLD, TRANSFER);
```

Comment:
Add parameterized test to verify that item is being transferred for both states.

Reply:
added